lblod/frontend-lokaal-mandatenbeheer
# Ben/lmb 433 installatievergadering routing

## Description

Updated the routing for the verkiezingen module, so this is a bit more clean. Added some helper functions to fetch bestuursorganen and updated the installatievergadering so there are multiple tables there for multiple bestuursorganen.
![Screenshot 2024-05-15 at 09 18 04](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/43848896/58e96c24-a3de-4351-9dde-72c082df5f84)
![Screenshot 2024-05-15 at 09 18 39](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/43848896/578e242f-b363-4de0-9a0f-9afedd20dd3f)

## How to test

!First update the backend and be sure the migrations have run.

Check out the verkiezingen route, see if everything is clear, if you find all relevant bestuursorganen in the installatievergadering (there should be six with burgemeester included) and test if everything still works.

## Remarks
There are still a few things that are a bit funky and should be fixed (there always is a burgemeester, whether you delete it or not, sorting is a bit broken and sorts all tables at once, the different tables should show different columns and tables should be autopopulated)

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/142
